x-pack/plugins/alerts/server/lib/license_state.ts

```diff
@@ -37,6 +47,47 @@ export class LicenseState {
     return this.licenseInformation;
   }
 
+  public setNotifyUsage(notifyUsage: LicensingPluginStart['featureUsage']['notifyUsage']) {
+    this._notifyUsage = notifyUsage;
+  }
+
+  public isLicenseValidForAlertType(
```

Comment:
nit.
Generally speaking I'd expecta method that starts with `is...` to return a boolean, while here we return a complex type.
It's no biggie, but I know this is a common pattern so I thought it's worth noting.